README.md

```diff
@@ -300,6 +301,36 @@ yarn dev
 # build the plugin for production to the `victoriametrics-logs-datasource` folder and zip build
 yarn build:zip
 ```
+#### 2. How to run a debugger for a backend plugin:
```

Comment:
I would suggest using this debug behavior 
https://st-g.de/2023/10/grafana-plugin-debugging-on-apple-silicon